openapi/task_execution_service.openapi.yaml

```diff
@@ -654,6 +671,14 @@ components:
           type: string
           description: |-
             Optional user-provided description of task for documentation purposes.
+        callback_url:
+          type: string
+          format: uri
+          description: |-
+            Optional user-provided callback URL. Note that the URL should be
+            provided while creating a new task, and it won't be displayed when
```

Comment:
Two minor wording issues:
> Note that the URL _can_ be provided (instead of: Note that the URL _should_ be provided)

> _but it MUST NOT_ be displayed (instead of: _and it won't_ be displayed)